src/plugins/intel_gna/tests/functional/pass_tests/convolution_concat_crop_memory.cpp

```diff
@@ -0,0 +1,135 @@
+            blob->allocate();
+
+            auto* rawBlobDataPtr = blob->buffer().as<float*>();
+            std::vector<float> values = CommonTestUtils::generate_float_numbers(blob->size(), -2.f, 2.f);
+            for (size_t i = 0; i < blob->size(); i++) {
+                rawBlobDataPtr[i] = values[i];
+            }
+            return blob;
+        }
+
+        void SetUp() override {
+            InferenceEngine::Precision netPrecision;
+            std::vector<size_t> inputShape = {1, 64};
+            std::tie(netPrecision, targetDevice, configuration, std::ignore) = this->GetParam();
+
+            auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+            size_t in_total_dims_size = ngraph::shape_size(inputShape);
+            auto params = ngraph::builder::makeParams(ngPrc, {{1, in_total_dims_size}});
+            auto pattern1 =
+                std::make_shared<ngraph::opset8::Constant>(ngraph::element::Type_t::i64, ngraph::Shape{2}, inputShape);
+            auto reshape1 = std::make_shared<ngraph::opset8::Reshape>(params[0], pattern1, false);
+            ngraph::VariableInfo vi{};
+            vi.data_shape = ngraph::PartialShape(inputShape);
+            vi.variable_id = "test_variable";
+            vi.data_type = ngraph::element::Type_t::f32;
+            const auto var = std::make_shared<ngraph::Variable>(vi);
+            std::vector<float> initial_state = CommonTestUtils::generate_float_numbers(in_total_dims_size, -3.f, 3.f);
+            auto initial_state_node =
+                ngraph::builder::makeConstant(ov::element::Type_t::f32, inputShape, initial_state);
+            auto readValue = std::make_shared<ngraph::opset9::ReadValue>(initial_state_node, var);
+            ngraph::OutputVector toConcat{readValue, reshape1};
+            const int axis = 1;
+            auto concat = ngraph::builder::makeConcat(toConcat, axis);
+
+            const auto concat_shape = concat->get_output_shape(0);
+            const auto concat_shape_size = ngraph::shape_size(concat_shape);
+
+            auto etlwise_data = CommonTestUtils::generate_float_numbers(concat_shape_size, -1.f, 1.f);
+            auto etlwise_node = ngraph::builder::makeConstant(ov::element::Type_t::f32, concat_shape, etlwise_data);
+            auto etlwise_result_node = std::make_shared<ngraph::opset9::Multiply>(concat, etlwise_node);
+
+            ngraph::ResultVector results{std::make_shared<ngraph::opset8::Result>(etlwise_result_node)};
+            auto split_node = ngraph::builder::makeSplit(concat, ngPrc, 2, axis);
+
+            auto assign_node = std::make_shared<ngraph::opset9::Assign>(split_node->output(1), var);
+            ngraph::SinkVector sinks{assign_node};
+            function = std::make_shared<ngraph::Function>(results, sinks, params, "CCCM");
+            ov::pass::Serialize("a1.xml", "a1.bin").run_on_function(function);
+        }
+};
+
+TEST_P(CCCMTest, CompareWithRefImpl) {
+    Run();
```

Comment:
Could we extend this test to run multiple iterations of inference (it is quite important since the model contains states)?